adal/build.gradle

```diff
@@ -67,6 +67,7 @@ android {
         disable 'GradleDependency'
         disable 'GradleOverrides'
         disable 'OldTargetApi'
+        disable 'PackageManagerGetSignatures'
```

Comment:
I don't actually see _any_ validation happening here. Is it just me, or does this method simply return whatever info.signatures[0] contains?